src/chaosinventory/authentication/serializers.py

```diff
@@ -0,0 +1,65 @@
+from django.contrib.auth import authenticate
+from rest_framework import serializers
+
+from .models import Token
+
+
+class ObtainAuthTokenSerializer(serializers.Serializer):
+    application = serializers.CharField(label="Application Name")
+    expiring = serializers.DateTimeField(
+        label="Date/Time when token expires",
+        required=False)
```

Comment:
The expiry date provided via the API can be in the past, I think if an application implements it it is their fault, but we could also throw an exception.